copier/main.py

```diff
@@ -1120,11 +1122,13 @@ def _apply_update(self) -> None:  # noqa: C901
                                 for line in conflicts_candidate
                             ):
                                 conflicted.append(fname)
-                    # Forcefully mark files with conflict markers as unmerged,
-                    # see SO post: https://stackoverflow.com/questions/77391627/
+                    # We ran `git merge-file` outside of a regular merge operation,
+                    # which means no merge conflict is recorded in the index.
+                    # Only the usual stage 0 is recorded, with the hash of the current version.
+                    # We therefore update the index with the missing stages:
+                    # 1 = current (before updating), 2 = base (last update), 3 = other (after updating).
+                    # See this SO post: https://stackoverflow.com/questions/79309642/,
```

Comment:
I think this comma is not needed, and it will help on some parsers that would take it as part of the URL:

```suggestion
                    # See this SO post: https://stackoverflow.com/questions/79309642/
```

Reply:
Oh, actually no... lets remove the period/comma then.